src/test/java/org/greenbuttonalliance/gbaresourceserver/usage/repository/CustomerRepositoryTest.java

```diff
@@ -0,0 +1,195 @@
+/*
```

Comment:
Looks like this entire file is a duplicate. Can you get rid of it?

Reply:
yes

Reply:
Actually, this is not a duplicate. CustomerRepositoryTest was already in Main. I'm not sure why, its not one I made. It came into my branch when I merged Main to it

Reply:
Although it is called CustomerRepositoryTest, the contents appear to be testing the IntervalBlock which makes this a duplicate file just called something else.

Reply:
I had it deleted, but was forced to add it back in when I merged main to my branch during the PR.